src/main/kotlin/com/atlassian/performance/tools/report/api/judge/Verdict.kt

```diff
@@ -14,15 +15,39 @@ class Verdict internal constructor(
     val reports: List<JUnitReport>,
     val failedActions: List<ActionType<*>>
 ) {
+    private val logger = LogManager.getLogger(this::class.java)
 
+    @Deprecated(message = "Use Builder instead", replaceWith = ReplaceWith("Verdict.Builder(reports).build()"))
     constructor(
         reports: List<JUnitReport>
     ) : this(
         reports = reports,
         failedActions = emptyList()
     )
 
-    private val logger = LogManager.getLogger(this::class.java)
+    /**
+     * Since 3.12.0
```

Comment:
Nit: We don't know about that yet 😉. I'm fine with this staying though.

Reply:
Those can be added in the "mark release" commits, but speculatively marking them here can also work

Reply:
In future changes I'm going to add similar changes at "mark release" stage